### DISCUSSION TITLE
为什么不把 repo transfor 到组织里，而是搞这么一通骚操作呢

### DISCUSSION THREAD

**Discussion by MistEO** [svc-develop-team/so-vits-svc#28] [General]
先是 fork，现在又直接上传所有文件。有没有考虑过 contributors 的感受？你们是 MIT 协议，还请考虑下 MIT 协议的宗旨是什么

**Comment by Miuzarte** [comment]
你说有没有可能旧的repo是直接被原作者删除了，我们只是在根据MIT协议来重新上传所有分支？

**Reply by MistEO** [reply]
原作者不是你们 org member？搁这儿唱双簧呢

**Reply by MistEO** [reply]
MIT 协议，好哇，今天我脸皮就厚一点，我那条 commit 的 contribution 呢？

**Reply by Miuzarte** [reply]
> MIT 协议，好哇，今天我脸皮就厚一点，我那条 commit 的 contribution 呢？

是我理解错了，transfor是真的做不到，当时r佬删库很突然
我电脑上还有部分分支的备份，可以尝试把这部分恢复上去（这个仓库第一个commit不是我的，我不背锅），但是你fork的仓库多了好几个分支，还是有部分commits估计恢复不了
![image](https://user-images.githubusercontent.com/66856838/226650599-9902c917-6fea-42e0-baa3-1b8c002ee1b7.png)
![image](https://user-images.githubusercontent.com/66856838/226650637-ce2cdcc9-dc97-4328-951e-02654c7d93bc.png)

**Reply by Miuzarte** [reply]
> MIT 协议，好哇，今天我脸皮就厚一点，我那条 commit 的 contribution 呢？

~~我v2的文件被新仓库覆盖过了，暂时只能恢复4.0分支的贡献~~
真要恢复的话重建仓库之后的贡献就没了（
要不在现在的README里我挂个贡献者列表吧？

**Comment by HuanLinOTO** [comment]


**Reply by MistEO** [reply]
所以谁付了我烟钱？另外不要偷换概念
